src/h5web/providers/mock/data.ts

```diff
@@ -415,10 +403,28 @@ export const mockMetadata = makeMetadata({
       [makeStrAttr('units', 'deg'), makeStrAttr('long_name', 'Angle (degrees)')]
     ),
     makeDataset('title_twoD', stringType, scalarShape),
-    makeSimpleDataset('errors_oneD', floatType, [41]),
     makeSimpleDataset('X_log', floatType, [41]),
     makeSimpleDataset('oneD_str', stringType, [2]),
     makeSimpleDataset('null', intType, []),
+    makeSimpleDataset(
+      'twoD_spectrum',
+      intType,
+      [20, 41],
+      [
+        makeStrAttr('interpretation', 'spectrum'),
+        makeStrAttr('units', 'arb. units'),
+      ]
+    ),
+    makeSimpleDataset('errors_twoD', floatType, [20, 41]),
+    makeSimpleDataset(
+      'NxFourD',
+      intType,
+      [3, 9, 20, 41],
+      [
+        makeStrAttr('long_name', 'Interference fringes'),
+        makeStrAttr('interpretation', 'image'),
```

Comment:
This fixes the misplaced `interpretation` attribute: it should be on the signal dataset and not on the NXdata group